### PR TITLE
Add OffsetDateTime support

### PR DESCRIPTION
**Pull request by koji-1009**

### CODE REVIEW

**Review by nacatl — APPROVED** [review]
LGTM

**Review by JASON13F — APPROVED** [review]
Nice!

**Review by JASON13F — APPROVED** [review]
Nice

File: studyplus-android-sdk/src/main/java/jp/studyplus/android/sdk/record/StudyRecord.kt [hunk 1]
```diff
@@ -20,14 +22,27 @@ data class StudyRecord @JvmOverloads constructor(
...
+        comment = comment,
+        recordedTime = recordedTime.toInstant().atOffset(ZoneOffset.UTC)
+    )
+
     internal fun toJson(): String = if (duration > DURATION_RANGE_MAX_24H) {
         throw IllegalArgumentException("duration must be 24 hours or less")
     } else {
         JSONObject().apply {
-            put("recorded_at", formatTime(recordedTime))
+            put("record_datetime", recordedTime.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME))
```
Comment (nacatl) [inline studyplus-android-sdk/src/main/java/jp/studyplus/android/sdk/record/StudyRecord.kt:45]: 👍

**Review by nacatl — APPROVED** [review]
LGTM！

**Pull request merged**